tensorflow/core/common_runtime/mkl_layout_pass_test.cc

```diff
@@ -1588,6 +1589,7 @@ REGISTER_TEST_ALL_TYPES(NodeRewrite_FusedConv2D_Positive1);
               "i:1, i:1} } }"                                                 \
               " attr { key: 'fused_ops'        value { list: {s: 'Relu'} } }" \
               " attr { key: 'epsilon'          value { f: 0.001 }}"           \
+              " attr { key: 'leakyrelu_alpha'  value { f: 0.2 }}"                    \
```

Comment:
Nit: would you mind moving the `\` at the end to align with the rest of the lines? Thank you!

Reply:
The `\` are aligned.